kfs-core/src/main/java/edu/arizona/kfs/tax/TaxConstants.java

```diff
@@ -0,0 +1,79 @@
+package edu.arizona.kfs.tax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaxConstants {
+
+
+    public static final String TAX_USER_ROLE = "UA 1099 Tax Specialist";
+    public static final String NMSPC_CD = "KFS-TAX";
+    public static final String REPORT_TEMPLATE_CLASSPATH = "com/rsmart/kuali/kfs/tax/report/";
```

Comment:
Still refers to legacy path for tax reports. Should this be addressed now or is that on a separate ticket? I notice that the files in this directory in KFS3 (ElectronicFilingReport.jrxml and ExceptionReport.properties) do not exist yet in ua-development and are not in this change set, so I'm assuming a separate ticket?